src/components/summary-list/index.md.njk

```diff
@@ -2,65 +2,130 @@
 
 If your summary list does not have any actions, you can choose to remove the separating borders with the `govuk-summary-list--no-border` class.
 
 {{ example({group: "components", item: "summary-list", example: "without-borders", html: true, nunjucks: true, open: false}) }}
 
 To remove borders on a single row, use the `govuk-summary-list__row--no-border` class.
 
+## Summary cards
+
+
+If you’re showing multiple summary lists on a page, you can show each list within a summary card. This lets you visually separate each summary list and give each a title and some actions.
+
+Use summary cards when you need to show:
+ - multiple summary lists that all describe the same type of thing, such as people
+ - actions that will apply to all the items in a list
+
+Summary cards are often used in case working systems to help users quickly view a set of information and related actions.
+
+Do not use summary cards if you only need to show a small amount of related information. Use summary lists instead, and structure them with headings if needed.
+
+If you’re showing summary cards at the end of a longer journey, you might want to familiarise the user with them earlier on — such as when the user reviews individual sections.
+
+### Card titles
+
+Use the summary card’s header area to give each summary list a title.
+
+Each title must be unique and help identify what the summary list describes. For example, this could be the name of a specific person, organisation or professional qualification.
+
+Try to keep titles short and relevant. You can use one or two important values in the summary list — such as the first and last name of a person.
+
+{{ example({group: "components", item: "summary-list", example: "card-with-title", html: true, nunjucks: true, open: false}) }}
+
+### Adding card actions
+
+You can add card actions in the header, which will be shown after the summary card's title.
+
+For example, if you have multiple rows with "change" actions that all take the user to the same place, you can show a single “change” card action instead. This helps avoid repeating the same row action on every row.
+
+Link text for card actions should tell the user what it will do and that it will apply to the entire in the summary card. It should also be as short as possible, usually 2 words.
+
+Example card actions include:
+
+ - Remove tenant
+ - Edit qualification
+ - Update issue
+ - Approve application
+ - Cancel order
+
+Keep it short and do not add more than 2 to 3 actions in a header.
+
+If a card action cannot easily be undone or might have serious consequences, consider adding a warning or asking the user for confirmation.
+
+{{ example({group: "components", item: "summary-list", example: "card-with-actions", html: true, nunjucks: true, open: false}) }}
+
 ## Research on this component
```

Comment:
@CharlotteDowns @paulrobertlloyd 

I'd like to commit some changes to the "Research" section (I got some comments that the bullet points could be clearer) and update "Next steps" with our current thinking. What do you think of this?

> ## Research on this component
> 
> This component was developed and tested by the Government Digital Service as part of the [check answers pattern](/patterns/check-answers/).
> 
> The Department for Education contributed the summary card. It’s being used in some of their services, such as:
> - [apply for teacher training](https://www.gov.uk/apply-for-teacher-training), used by the general public
> - [register trainee teachers](https://www.register-trainee-teachers.service.gov.uk/), used by people that work for training providers
> 
> The summary card is also used in services run by other departments, such us:
> - manage supervisions (Ministry of Justice)
> - submit social housing lettings and sales data (Department for Levelling Up, Housing & Communities)
> 
> ### Next steps
> 
> We still want to learn more about the types of services and information this component works well with. If you use this component in your service, we'd like to hear about you use this components and any research findings you might have.

Reply:
Looks good to me.

Reply:
Hey @calvin-lau-sig7 I was just reviewing the 'next steps' section in the most recent [preview](https://deploy-preview-2251--govuk-design-system-preview.netlify.app/components/summary-list/#next-steps), and I wondered if there might be a missing reference to 'summary card':

**Current:**

> If you use this component in your service, we'd like to hear about how you use the summary list and summary, as well as any research findings you might have.

**Is it meant to say:**

> If you use this component in your service, we'd like to hear about how you use the summary list and summary **card**, as well as any research findings you might have.

As far as I can tell, we don't have a style, component, or pattern that we refer to as just 'summary', hence why I got curious. I suppose it could be referring to people using the `<summary>` tag though.

Reply:
You're right there! I was flip-flopping on how to describe the 2 components (or variants) and went with the easy route of calling them by name.